app/models/mixins/supports_feature_mixin.rb

```diff
@@ -74,6 +74,14 @@ module SupportsFeatureMixin
   # Whenever this mixin is included we define all features as unsupported by default.
   # This way we can query for every feature
   included do
+    if self.class == Module
+      raise "SupportsFeatureMixin cannot be included into modules. You have to use 'extend ActiveSupport::Concern'"
```

Comment:
:smile: yea, ack this reads awkward, but I couldnt come up with a better oneliner to explain [this documentation](https://github.com/ManageIQ/manageiq/blob/e61dbc4ffb26158a1e4603a93feef81a9f27ea39/app/models/mixins/supports_feature_mixin.rb#L38-L51)

But maybe this whole PR isnt needed at all because I suck at inclusion